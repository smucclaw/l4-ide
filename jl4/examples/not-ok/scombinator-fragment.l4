ยง `The S combinator`

GIVEN t1 IS A TYPE
      t2 IS A TYPE
      t3 IS A TYPE
      f IS A FUNCTION FROM t1 AND t2 TO t3
      g IS A FUNCTION FROM t1 TO t2
GIVETH A t3
s f g x MEANS
  f OF
