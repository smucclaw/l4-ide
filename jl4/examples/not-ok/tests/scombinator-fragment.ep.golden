2025-03-12T18:00:01.103797Z | Info | updateFileDiagnostics published different from new diagnostics - file diagnostics:
File:    
    file:///home/mangoiv/Devel/well-typed/cclaw/l4-ide/jl4/examples/not-ok/scombinator-fragment.l4
  Hidden:   no
  Range:    11:1-11:1
  Source:   parser
  Severity: DiagnosticSeverity_Error
  Code:     <none>
  Message:  [0;91m|11 || ^incorrect indentation (got 1, should be greater than 3)[0m